assets/keymaps/linux/jetbrains.json

```diff
@@ -59,7 +59,9 @@
       "ctrl-shift-end": "editor::SelectToEnd",
       "ctrl-f8": "editor::ToggleBreakpoint",
       "ctrl-shift-f8": "editor::EditLogBreakpoint",
-      "ctrl-shift-u": "editor::ToggleCase"
+      "ctrl-shift-u": "editor::ToggleCase",
+      "ctrl-shift-up": "editor::MoveLineUp",
+      "ctrl-shift-down": "editor::MoveLineDown"
```

Comment:
Just to note - JetBrains has two different ways to do this, there's

`ctrl-shift-up|down` which moves expression up/down logically, in other words, it operates on syntax tree (e.g. triggering moving line up on method name moves the whole method)

`alt-shift-up|down` which move line up/down and operates on raw file lines, disregarding syntax

I'm not sure which Zed MoveLine actions match this behaviour, but I'd guess it's the latter?

Reply:
Intellij calls these actions - `Move Statement`:

![image](https://github.com/user-attachments/assets/46ff767c-0656-421c-afbb-7ce7eceb2ea0)

and `Move Line`:

![image](https://github.com/user-attachments/assets/921c8745-89e2-43ad-bc49-0e7bb3f034f3)